### PR TITLE
ENH: Gallery, various little stylish corrections (DFT example).

### PR DESCRIPTION
**Pull request by sciunto**
Just to make things nicer...

### CODE REVIEW

File: doc/examples/transform/plot_register_translation.py [hunk 1]
```diff
@@ -39,16 +38,16 @@
...
 
-ax2.imshow(offset_image.real)
+ax2.imshow(offset_image.real, cmap='gray')
 ax2.set_axis_off()
 ax2.set_title('Offset image')
 
-# View the output of a cross-correlation to show what the algorithm is
-#    doing behind the scenes
+# Show the output of a cross-correlation to show what the algorithm is
+# doing behind the scene
```
Comment (soupault) [inline doc/examples/transform/plot_register_translation.py]: Wasn't there an idiom (`behind the scenes`)?

Comment (alexandrejaguar) [inline doc/examples/transform/plot_register_translation.py]: I think so. Maybe `on the background` would be more appropriated?

**Comment by sciunto** [comment]
Fixed.

**Comment by soupault** [comment]
Thanks @sciunto !

**Pull request merged**